aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -147,13 +263,15 @@ module aptos_framework::staking_config {
         staking_config.recurring_lockup_duration_secs = new_recurring_lockup_duration_secs;
     }
 
+    /// DEPRECATING
     /// Update the rewards rate.
     /// Can only be called as part of the Aptos governance proposal process established by the AptosGovernance module.
     public fun update_rewards_rate(
         aptos_framework: &signer,
         new_rewards_rate: u64,
         new_rewards_rate_denominator: u64,
     ) acquires StakingConfig {
+        assert!(!features::reward_rate_decrease_enabled(), EDEPRECATED_FUNCTION);
```

Comment:
nit: Use error::some_appropriate_function to wrap the error code. Here and in other places below